src/early_multicellular_stage/editor/EarlyMulticellularEditor.cs

```diff
@@ -79,7 +79,7 @@ public override bool CanCancelAction
     public Patch? SelectedPatch => patchMapTab.SelectedPatch;
 
     [JsonIgnore]
-    public ICellProperties? EditedCellProperties => selectedCellTypeToEdit;
+    public ICellDefinition? EditedCellProperties => selectedCellTypeToEdit;
```

Comment:
EditedCellProperties could also be renamed, not necessarily though

Reply:
I was thinking about that, but as goes so high up the chain that at that level there doesn't really logically exist a cell definition that is being modified, but just the general properties of a thing that is being modified. I know that's not the best explanation but when I did initially rename this property as well, it didn't really fit in my head as to what the editor does, so I left it unused. This shouldn't really be able to cause confusion as there doesn't exist a gameplay simulation in the editor that would have `Components.CellProperties` that could be confused with this.